### PR TITLE
Chore: salesforce remediate vulnerable deps [integ-2287]

### PR DESCRIPTION
**Pull request by sarahlessner**

### CODE REVIEW

**Review by david-shibley-contentful — APPROVED** [review]

File: apps/salesforce-commerce-cloud/package.json [hunk 1]
```diff
@@ -14,8 +15,8 @@
     "@tanstack/react-query": "^4.29.7",
     "@vitejs/plugin-react": "^4.3.1",
     "axios": "^1.7.7",
-    "commerce-sdk": "^2.10.1",
-    "contentful-management": "10.7.0",
+    "commerce-sdk": "^2.15.0",
+    "contentful-management": "^11.35.1",
```
Comment (mgoudy91) [inline apps/salesforce-commerce-cloud/package.json:19]: might cause a problem again, we'll see

**Review by mgoudy91 — COMMENTED** [review]
code looks totally fine, only concern is the cma v11 update, but thats why we have staging envs eventually

File: apps/salesforce-commerce-cloud/package.json (hunk 1 above)
Comment (sarahlessner) [reply apps/salesforce-commerce-cloud/package.json:19]: Curious do we know what specific problem did this library update cause in the past? I think I've merged in a few with the update to 11 and they were still working in prod. 

I am working on getting staging setup for this one. I got a staging environment set up and deployed the bundle but ran out of time to go through the config. I'll make sure that's done and tested before I merge.

Comment (sarahlessner) [reply apps/salesforce-commerce-cloud/package.json:19]: I can also update this to the earliest version with a fix (assuming that's even an option) if it's safer.

**Review by david-shibley-contentful — APPROVED** [review]

**Pull request merged**